pkg/discovery/dtofactory/property/property_common.go

```diff
@@ -21,6 +21,7 @@ const (
 	k8sAppType                   = "KubernetesAppType"
 	TolerationPropertyNamePrefix = "[k8s toleration]"
 	LabelPropertyNamePrefix      = "[k8s label]"
+	k8sVolumeCount               = "PersistentVolumeAttachedCount"
```

Comment:
My initial thought around this was to have a boolen property, ie `volumeAttached=true/false`. This provides more fine grained information.
The tags however are used in the UI to be able to filter the entities based on these tags. Please test it out in the UI to see if the filter as count are meaningful and can be used to actually create the groups, which is the main intention of introducing this property.

Reply:
@irfanurrehman  please have a look at my test through UI. I put some screenshots in the description.

Reply:
I was also thinking of having a boolean initially.

Reply:
It is ok to have a count so long as one can easily create a group for PersistentVolumeAttachedCount > 0.  If not, then you may want to have a boolean also.

Reply:
@pankajbatra1  @irfanurrehman  I changed to use the `boolean` type for the value, does this looks good to you?

Reply:
@kevinwangcn I think this code is allright. Just ensure that we can filter the pods using this property.
Approving with this change

Reply:
Thanks @irfanurrehman